filament/include/filament/Camera.h

```diff
@@ -350,6 +378,25 @@ class UTILS_PUBLIC Camera : public FilamentAPI {
     void setModelMatrix(const math::mat4& model) noexcept;
     void setModelMatrix(const math::mat4f& model) noexcept; //!< @overload
 
+    /** Set the position of each eye relative to this Camera (head).
+     *
+     * By default, these are both identity matrices.
+     *
+     * For example, to position Eye 0 3cm leftwards and Eye 1 3cm rightwards:
+     * ~~~~~~~~~~~{.cpp}
+     * mat4 eyes[2];
+     * eyes[0] = mat4::translation(double3{-0.03, 0.0, 0.0});
+     * eyes[1] = mat4::translation(double3{ 0.03, 0.0, 0.0});
+     * camera.setEyeModelMatrix(eyes);
+     * ~~~~~~~~~~~
+     *
+     * This method is not intended to be called every frame. Instead, to update the position of the
+     * head, use Camera::setModelMatrix.
+     *
+     * @param model an array of two eye model matrices, the first, Eye 0, the second, Eye 1.
+     */
+    void setEyeModelMatrix(math::mat4 const (&model)[2]) noexcept;
```

Comment:
I would prefer an API that takes an "eye id" instead of an array. e.g.:

`setEyeModelMatrix(uint8_t eyeId, mat4& model)`